src/requirements/data/colleges/en.ts

```diff
@@ -105,61 +106,73 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
           distribution => course.catalogDistr?.includes(distribution) ?? false
         ),
     ],
-    checkerWarning: 'We do not check that at least two courses are at the 2000 level or higher.',
+    checkerWarning:
+      'We do not check that all categories are being met and that at least two courses are at the 2000 level or higher.',
     fulfilledBy: 'courses',
     perSlotMinCount: [6],
   },
+  // TODO: Spliting Liberal Studies into 3 parts means they can all never be
```

Comment:
The liberal studies req couldn't currently be fulfilled (bc it would require triple counting). Splitting it into three requirements also meant that a lib study course couldn't meet other requirements too. So for now, just made it one requirement with a warning

Reply:
It can be fulfilled. Screenshot from master.
<img width="1004" alt="Screen Shot 2021-03-29 at 13 52 36" src="https://user-images.githubusercontent.com/4290500/112878517-0c40ce80-9096-11eb-8eb9-fb4b8296701e.png">
I think what we can do is to add another requirement that checks 2000 level or higher part. Then the warning can be removed. Then with the change in #401, it will be selected by default so user won't be confused.